src/Dropzone/assets/src/style.css

```diff
@@ -60,7 +60,7 @@
 }
 
 .dropzone-preview-button::before {
-    content: '×';
+    content: "×";
```

Comment:
Let's remove this :) 

We'll make the change in another small PR

Reply:
I never made that change, I think that's just the linter ?